src/PHPCensor/Model/Build.php

```diff
@@ -165,6 +168,11 @@ class Build extends Model
             'nullable' => true,
             'default'  => null,
         ],
+        'environment' => [
+            'type'    => 'varchar',
+            'length'  => 20,
```

Comment:
I would make a limit of 250 (like a 'branch' field in the 'build' table). 20 is too short